src/document/DocumentAnnotator.ts

```diff
@@ -38,8 +38,8 @@ export default class DocumentAnnotator extends BaseAnnotator {
 
         // Lazily instantiate managers as pages are added or re-rendered
         if (managers.size === 0) {
-            managers.add(new RegionManager({ location: pageNumber, referenceEl: pageReferenceEl }));
             managers.add(new HighlightManager({ location: pageNumber, referenceEl: pageReferenceEl }));
+            managers.add(new RegionManager({ location: pageNumber, referenceEl: pageReferenceEl }));
```

Comment:
This is so that the highlight layer will be on top of the region layer (due to the way we insert today) so that highlights are clickable above regions

Reply:
Can you confirm with product? I remember regions should be above highlights.

Reply:
Yeah, I checked with product and design